cmd/server/main.go

```diff
@@ -58,6 +56,24 @@ func main() {
 	defer logger.Sync()
 	logger.Infof("%s", version.Long)
 
+	if opt.SignalUpgrade {
```

Comment:
So this option provides a manual way for sending the USR2 signal to the original Easegress process? It got me thinking first. Maybe we can leave some comments here.